lib/param_parsing/parameters.go

```diff
@@ -56,19 +56,52 @@ func GetParameters() Params {
 	logger = lib.InitLogger(params.LogLevel)
 	var err error
 	// Read configuration files
-	if tomlFileExists(params) {
-		params, err = getParamsTOML(params)
-	} else if tfSwitchFileExists(params) {
+	// TOML from Homedir
+	if tomlFileExists(lib.GetHomeDirectory()) {
+		params, err = getParamsTOML(params, lib.GetHomeDirectory())
+		if err != nil {
+			logger.Fatalf("Failed to obtain settings from TOML config in home directory: %q", err)
+		}
+	}
+
+	// TOML from ChDirPath
```

Comment:
Based on https://github.com/warrensbox/terraform-switcher/discussions/420#discussioncomment-9369690 do we really want TOML in ChDirPath? 🤔

Reply:
I guess we could do with some weight in from @warrensbox , as this seemed quite specific in the main description:
```
But what is the .toml file is not in the HOME directory. That's where the option to pass the -c came in.

SO we still need to make the toml file readable as well incorporating the ability to have the version overridden.
```

Reply:
👍🏻 
I personally see no reason for TOML to exist out of HOME (or XDG Config) dir since this adds ambiguity as TOML defines not just version parameter.
Referencing back to you https://github.com/warrensbox/terraform-switcher/discussions/420#discussioncomment-9371421 also =)
There are other methods to define per-dir (incl `ChDirPath`) TF version and I'd prefer TOML to exist only in HOME (or XDG) to eliminate any kind of confusion that may add complexity understanding where the hiccup came from for users and for us when debugging issues.

Reply:
Here are some options:

@yermulnik @MatthewJohn 
- users cannot specify a **version** in the TOML file. The TOML is used for all other configurations except **version**
- only the TOML file in HOME can have a version (which later can be overriden)

Thoughts?

Reply:
@yermulnik the TOML file is useful mostly for including the bin location. 
`bin: <path-to-install-tf>
If the user does not have access to the HOME directory, how would they specify the `bin` directory?

What is your suggestion?
- TOML only available in $HOME
- if TOML is used in ChDirPath, they can only include version ?

Reply:
>Though, I can imagine any changes to these could be fairly catastrophic - tfswitch provides a lot of ways of passing in versions and you could almost guarantee that each of the methods is used by at least one person :D

Haha yes this has always been the case - “The Needs of the Many Outweigh the Needs of the Few” 🖖

Reply:
> the TOML file is useful mostly for including the bin location.

Also `log-level` (which is not working properly when defined with TOML file at the moment, though hopefully this PR will get this fixed 🤞🏻) and potentially other features we may add in future.

> What is your suggestion?

The «_TOML only available in $HOME_» is my pref.
The «_if TOML is used in ChDirPath, they can only include version ?_» is what I can accept if others think this option makes sense.

Reply:
One idea I did have is to completely ignore any TOML files in Chdir but instead to add a variable for TOML file location, which defaults to the homedir TOML path.
This would add the logical separation:
 * `Chdir` is to specify the location of your _code_ which tfswitch will check for versions
 * `TomlFile` is to simply change where the TOML file can is

We would only attempt to read the TOML file once (in the Tomlfile param path)

Reply:
@yermulnik why can't the toml file in `ChDirPath` include `bin` as well?
I might be missing something here.

Reply:
Actually I don't see a common use case for the TOML file to reside in `ChDirPath`. IMHO this would add complexity investigating issues and make debugging convoluted. I mean exactly non-`version` configuration parameters from TOML file.
If users need to override `bin` or `log-level` per use case, then they can (and should) use cmdline options. IMHO.
My vote goes for **explicit** (cmdline args) rather than **implicit** (TOML in `ChdriPath`).

Along with that what @MatthewJohn suggests above re the option to redefine TOML file location using cmdline option, absolutely makes sense to me and is a pretty common approach.

ps: I'm not pushing my opinion as an absolute one. If majority of us see TOML in `ChDirPath` as a good option, I'm fine with it. My only intention is to reduce failure and complexity points 🤝

Reply:
From the comment in #420 from Warren, that we should no longer support ChDirPath TOML file.. I've realised that the tests around the TOML file make heavy use of this - meaning that the only way we can test integration with the TOML file is to modify the TOML file in homedir - which isn't great for running tests.
Any suggestions for this? (I could leave the Toml directory argument in `getParamsTOML`, allowing the tests to test `getParamsTOML` passing in a test directory - but not sure this is fantastic

Reply:
Any concerns with the changes? If not, I'll resolve the comment :)